Source/fontconfig/FCFontEnumerator.m

```diff
@@ -189,23 +391,27 @@ - (void) enumerateFontsAndFamilies
 
               familyString = [NSString stringWithUTF8String: family];
               familyArray = [fcxft_allFontFamilies objectForKey: familyString];
+
               if (familyArray == nil)
                 {
                   NSDebugLLog(@"NSFont", @"Found font family %@", familyString);
-                  familyArray = [[NSMutableArray alloc] init];
```

Comment:
I prefer to have non-autorelease objects especially if they are long lived (but also for the very short lived ones).

Reply:
I'm not sure what you're referring to here -- familyArray _isn't_ autoreleased.

Reply:
Oh, I see what you are talking about now. The location at which you made your comment threw me off.